demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -3501,9 +3569,6 @@ export class DemoMeetingApp
 
   remoteVideoSourcesDidChange(videoSources: VideoSource[]): void {
     this.log(`available remote video sources changed: ${JSON.stringify(videoSources)}`);
-    if (!this.enablePin) {
```

Comment:
So originally I did this so that we have a way to test the default behavior for priority-based downlink (we accidentally broke that before because we have no way to test it in this demo). Maybe we can add a boolean flag here to only do this if users click on target size or priority first.

Reply:
I have refactored it in a way where this is still possible as mentioned in desc.  Before you press any config, it will never call `chooseRemoteVideos` because it will short circuit if `this.attendeeIdToVideoPreference` is empty (tested this as well).

Reply:
Oh I missed that. Thanks